Sources/VaporAWSLambdaRuntime/LambdaServer.swift

```diff
@@ -10,13 +10,18 @@ public protocol VaporLambda: ByteBufferLambdaHandler {
     var app: Application { get }
     static var requestSource: RequestSource { get }
 
+    /// Used to set up globals such as logging, tracing, metrics, etc.
+    static func setup() async throws
```

Comment:
I'm not 100% sure about the name `setup()` – from the naming alone, it's not immediately clear that this runs before creating the application. Maybe something like `bootstrap()` works better?